src/main/scala/io/opentargets/etl/backend/literature/Literature.scala

```diff
@@ -41,6 +41,8 @@ object Literature extends LazyLogging {
     Embedding()
     logger.info("Run literature vectors")
     Vectors()
+    logger.info("Run literature vectors")
```

Comment:
Got you @remo87 ! hahaha (evil laugh)!
Thanks a lot for the PR, great work!
Please, fix these logging messages to remove duplication and make them more informative and re-request the PR review.

Thanks!

Reply:
Thanks! just fixed it.